src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -193,6 +193,10 @@ public bool PrimaryNodeIsRequired
         /// Whether to use the gossip optimization to speed up spreading liveness information.
         /// </summary>
         public bool UseLivenessGossip { get; set; }
+        /// <summary>
+        /// Whether new silo that joins the cluster has to validate the initial connectivity with all other Active silos.
```

Comment:
How about: set this to true if you are running in unreliable Cloud Provider that does not guarantee basic network connectivity.
Set this to false for any other, normal Cloud Provider.